### ISSUE TITLE
Unable to run the site locally with node 6.10.2

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/shakespeares_world#333]
I don't know if this is related to #332 but I cannot get SW to run locally either. `npm start` gives a bunch of scary looking C stack traces from the v8 javascript engine.
```
==== C stack trace ===============================

 1: v8::Template::Set(v8::Local<v8::Name>, v8::Local<v8::Data>, v8::PropertyAttribute)
 2: fse::FSEvents::Initialize(v8::Local<v8::Object>)
 3: node::DLOpen(v8::FunctionCallbackInfo<v8::Value> const&)
 4: v8::internal::FunctionCallbackArguments::Call(void (*)(v8::FunctionCallbackInfo<v8::Value> const&))
 5: v8::internal::(anonymous namespace)::HandleApiCallHelper(v8::internal::Isolate*, v8::internal::(anonymous namespace)::BuiltinArguments<(v8::internal::BuiltinExtraArguments)3>)
 6: v8::internal::Builtin_Impl_HandleApiCall(v8::internal::(anonymous namespace)::BuiltinArguments<(v8::internal::BuiltinExtraArguments)3>, v8::internal::Isolate*)
 7: v8::internal::Builtin_HandleApiCall(int, v8::internal::Object**, v8::internal::Isolate*)
 8: 0x1288111092a7
 9: 0x1288117be944
10: 0x128811508718
```
node: 6.10.2
npm: 3.10.10
OS: OSX 10.11.6

**Comment by simoneduca** [comment]
TL;DR
The problem seems caused by a bunch of packages needing updating and some further fiddling. 
[Here's the PR](https://github.com/zooniverse/shakespeares_world/pull/334).

This is how I managed to get it working with latest node and all previous versions:

1. `npm-check-updates -u`
    `npm install`

2. Got this 
![screen shot 2017-08-21 at 22 45 17](https://user-images.githubusercontent.com/8989606/29539606-9853f0aa-86c2-11e7-8253-649890c0c405.png) so changed the name `angular-ui-router` to `@uirouter/angularjs`. Make sure you update the relevant import in `app.module.js`

3. Got error on `npm start`:  `Error: Cannot find module 'jshint/src/cli`. Installed it like this `npm install --save-dev jshint gulp-jshint`, because of [changes to how internal dependencies were installed in that package](https://github.com/spalger/gulp-jshint/issues/131).

 4. At this stage, the code compiles correctly. Except that the local server stops after running the Gulp `clean` task. No idea why, but if you removed the task from the `build` task, which is turn called by `development`, all is good.

**Comment by eatyourgreens** [comment]
`npm i -g npm-check-updates` to install that first tool?

**Comment by simoneduca** [comment]
Yep, thanks. I had it already and forgot to mention it.

**Comment by eatyourgreens** [comment]
:+1: the app starts now without any stack traces. All I get in the browser is a blank grey page, which I think is the issue reported in #332 

I'm starting the app using `NODE_ENV=production npm start`.

**Comment by simoneduca** [comment]
I have no idea... I can run the app locally and against production, with node 6.10 and latest 8.2.

**Comment by eatyourgreens** [comment]
Nothing here, just the background colour and an empty DOM. I'm sure this is the same issue that's reported in #332, so I won't open a new issue.
